src/components/Main.tsx

```diff
@@ -1,16 +1,18 @@
-import { Flex, Text } from '@chakra-ui/react'
+import { Flex } from '@chakra-ui/react'
+
+import Login from './Login'
```

Comment:
Para importar um **componente** pode usar o caminho `components/SeuComponente`
```suggestion
import Login from 'components/Login'
```